src/main/js/field/PopUp.js

```diff
@@ -1,69 +1,52 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Modal } from 'react-bootstrap';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import * as FieldModelUtilities from 'util/fieldModelUtilities';
-import FieldsPanel from './FieldsPanel';
+import ConfigButtons from 'component/common/ConfigButtons';
 
 class PopUp extends Component {
     constructor(props) {
         super(props);
 
         this.internalCancel = this.internalCancel.bind(this);
-        this.internalOk = this.internalOk.bind(this);
-
-        this.state = {
-            modalConfig: {},
-            fieldErrors: {}
-        };
     }
 
     internalCancel() {
         this.props.onCancel();
-        this.setState({
-            modalConfig: {}
-        });
-    }
-
-    internalOk(modalConfig) {
-        this.props.onOk(modalConfig);
-        this.internalCancel();
     }
 
     render() {
         const {
-            fields, show, title, cancelLabel, okLabel
+            children, show, title, cancelLabel, okLabel, handleSubmit, performingAction
         } = this.props;
-        const { modalConfig, fieldErrors } = this.state;
-
         return (
             <div>
-                <Modal show={show} onHide={this.internalCancel}>
+                <Modal size="lg" show={show} onHide={this.internalCancel}>
```

Comment:
Can't you use the modified PopUp component here?

Reply:
This is defining the modified PopUp component